variables.tf

```
@@ -94,3 +94,9 @@ variable "ip_protocol" {
   description = "The IP protocol for the backend and frontend forwarding rule. TCP or UDP."
   default     = "TCP"
 }
+
+variable "service_label" {
+  description = "Service label is used to create internal DNS name"
+  default     = "srv"
```

Let's default this to null to preserve the existing behaviour.

```suggestion
  default     = null
  type        = string
```